Gemfile

```diff
@@ -1,7 +1,4 @@
 source 'https://rubygems.org'
 gemspec
 
-group :compilation do
```

Comment:
@MSP-Greg : same here. Can you please revert this?
I will clean this up later. I am still situating myself and making sure I situate myself before starting deleting and cleaning up.